commons/_shared/bs-lib-components/ui/dist/types/components/table/BsDSSTableFunctional.vue.d.ts

```diff
@@ -23,12 +24,13 @@ declare const _sfc_main: import("vue").DefineComponent<{
     setFetchingSchema(fetchingSchema: boolean): void;
     setFetchingChunk(fetchingChunk: boolean): void;
     fetchDSSData(datasetName: string, chunksize?: number | undefined, chunkIndex?: number | undefined): Promise<Record<string, any>[] | undefined>;
+    fetchFilteredDSSDataset(datasetName: string, chunksize: number, chunk_index: number, filters?: Record<string, any[]> | undefined): Promise<Record<string, any>[] | undefined>;
     fetchDSSColumns(datasetName: string): Promise<{
         columns: BsTableCol[];
         columnsCount: number;
     }>;
     updateColumns(datasetName: string): void;
-    updateRows(datasetName: string, chunksize?: number | undefined, chunkIndex?: number | undefined): void;
+    updateRows(datasetName: string, chunksize: number, chunk_index: number, filters?: Record<string, any[]> | undefined): void;
```

Comment:
filters should have a type of `Records<string, string[]>`

Reply:
cant we have numbers for exp
?